Packaging.props

```diff
@@ -13,6 +13,7 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == ''">$(Platform)</PackagePlatform>
     <PackagePlatform Condition="'$(PackagePlatform)' == 'amd64'">x64</PackagePlatform>
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
+    <XmlDocFileRoot>$(PackagesDir)Microsoft.Private.Intellisense/1.0.0-rc4-24206-00/xmldocs</XmlDocFileRoot>
```

Comment:
Is the plan just to leave this hard-coded at rc4-24206-00?  Otherwise, I'd think we'd want the version for this to be an additional property which we update from the same place we update other versions instead of tracking it down in packaging.props when it needs to be updated..